### PR TITLE
SPARK-6626 [DOCS]: Corrected Scala:TwitterUtils parameters

### PR DESCRIPTION
**Pull request by VeylanSolmira**
Per Sean Owen's request, here is the update call for TwitterUtils using Scala :)

### CODE REVIEW

File: docs/streaming-programming-guide.md [hunk 1]
```diff
@@ -704,7 +704,7 @@ create a DStream using data from Twitter's stream of tweets, you have to do the
 {% highlight scala %}
 import org.apache.spark.streaming.twitter._
 
-TwitterUtils.createStream(ssc)
+TwitterUtils.createStream(ssc, None)
```
Comment (JaysonSunshine) [inline docs/streaming-programming-guide.md]: Without a second parameter of type Option, createStream requires ssc to be a jssc object.

**Comment by srowen** [comment]
LGTM, will merge after a short time. I will run the doc build manually to double-check.

**Comment by srowen** [comment]
PS Usually you make the title like `SPARK-6626 [DOCS] ...`, could you update it?
https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark

**Comment by JaysonSunshine** [comment]
Okay, updated. The system had added 5295 to my commit so I'm assuming that is the number to use :)

**Comment by rxin** [comment]
Actually this is not the right JIRA ticket. https://issues.apache.org/jira/browse/SPARK-5295 is about data types.

**Pull request closed without merging**